mParticle_Optimizely/MPKitOptimizely.m

```diff
@@ -81,46 +52,17 @@ - (MPKitExecStatus *)didFinishLaunchingWithConfiguration:(NSDictionary *)configu
         return [self execStatus:MPKitReturnCodeRequirementsNotMet];
     }
     
-    NSNumber *eventInterval = self.configuration[oiEventInterval];
-    NSNumber *dataFileInterval = self.configuration[oiDataFileInterval];
-    
     _configuration = configuration;
     
     if (MPKitOptimizelyClient == nil) {
-        static dispatch_once_t optimizelyPredicate;
+        MPKitOptimizelyClient = [MPKitOptimizelyClient initWithSdkKey:sdkKey];
```

Comment:
I guess this is pre-existing but it seems pretty weird to me to have an uppercase first letter for a variable, typically I would expect that to represent a class name.